config/version-number-rules.xml

```diff
@@ -32,5 +32,11 @@
         <ignoreVersion type="regex">.*</ignoreVersion>
       </ignoreVersions>
     </rule>
+    <rule groupId="com.github.spotbugs" artifactId="spotbugs-maven-plugin">
+      <ignoreVersions>
+        <!-- util https://github.com/spotbugs/spotbugs/issues/1601 -->
```

Comment:
util -> until
please fix this typo for "check closed issue refs job" to work properly

Reply:
done